net-libs/libblkmaker/libblkmaker-0.5.3.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="A library implementing getblocktemplate"
+DESCRIPTION="libblkmaker provides an implementation of getblocktemplate, the\
+ current Bitcoin mining protocol."
+HOMEPAGE="https://github.com/bitcoin/libblkmaker"
+COPYRIGHT="Luke Dashjr"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/bitcoin/libblkmaker/archive/v${portVersion}.tar.gz"
+CHECKSUM_SHA256="549193640d4cc7022725754833d9d8a7639c41e1a7d2c5f2588be65857cc0970"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
```

Comment:
It might be better to mark x86_gcc2 as untested:
```diff
-ARCHITECTURES="x86_gcc2 x86 x86_64"
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
```
as libblkmaker needs libbase58 which does not build on x86_gcc2 primary arch without a patch.
(Yes, I know you wrote one, and I even tested it and found that it works like a charm ;-)